package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "dependencies": {
     "govuk-frontend": "^3.5.0",
+    "govwifi-shared-frontend": "https://github.com/alphagov/govwifi-shared-frontend/releases/download/0.6.2/govwifi-shared-frontend-0.6.2.tgz",
```

Comment:
We should update the version here as well (and the package lock file).

Reply:
Yeah for some reason I'd only committed the `.lock` file change, which is strange....